core/src/main/java/org/opensearch/sql/expression/function/BuiltinFunctionName.java

```diff
@@ -149,6 +149,8 @@ public enum BuiltinFunctionName {
   STDDEV_POP(FunctionName.of("stddev_pop")),
   // take top documents from aggregation bucket.
   TAKE(FunctionName.of("take")),
+  // TODO: nested aggregation function
```

Comment:
Not sure if we wanted to remove the TODO?
```suggestion
  //nested aggregation function
```

Reply:
No, this should remain as it is not implemented yet.